geode-core/src/main/java/org/apache/geode/internal/cache/control/ResourceManagerStats.java

```diff
@@ -92,6 +95,16 @@
             f.createLongCounter("rebalanceTime",
                 "Total time spent directing cache rebalance operations.", "nanoseconds", false),
 
+            f.createLongCounter("restoreRedundanciesInProgress",
+                "Current number of cache restore redundancy operations being directed by this process.",
+                "operations"),
+            f.createLongCounter("restoreRedundanciesCompleted",
+                "Total number of cache restore redundancy operations directed by this process.",
+                "operations"),
+            f.createLongCounter("restoreRedundancyTime",
+                "Total time spent directing cache restore redundancy operations.", "nanoseconds",
+                false),
+
```

Comment:
I think we should add these new statistics to `geode-docs/reference/statistics_list.html.md.erb`.

Reply:
Thanks for pointing this out, I was unaware of this doc. I'll add the changes to the final PR for this set of changes, which will include all docs updates.